src/logoutd.c

```diff
@@ -15,6 +15,7 @@
 #include <stdio.h>
 #include <sys/stat.h>
 #include <sys/types.h>
+#include <utmp.h>
```

Comment:
Huh, weren't we going to get rid of logoutd? :)

Reply:
Yes, but the compilation was failing because the `include` was missing and this seemed the easier and fastest approach to fixing it.